### ISSUE TITLE
Add backdrop art options to GUI

### ISSUE THREAD

**Issue by raphkoster** [libretro/mame2003-plus-libretro#706]
Now that backdrops work, it would be nice to add the options that existed originally to display with and without bezels, etc, to the GUI and the core options. With most libretro frontends, adding command line args is difficult to impossible.

From the original implementation in MAME 0.61 (see https://wiki.mamedev.org/index.php/MAME_0.61):

    -noartcrop -- crops artwork to the game screen area only
    -nobackdrop -- enables/disables backdrop art
    -nooverlay -- enables/disables overlay art
    -nobezel -- enables/disables bezel art

This would enable people to use the support for .art files in tandem with the usual overlays.

**Comment by Riddler1** [comment]
Yes, please I was going to request if overlays could be added for use like they are Mame 2010 that would allow Space Invaders for example to have it's Red and Green colors at the top and bottom of the screen repectively.
